### ISSUE TITLE
Test breaks and page freezes on axe-beta

### ISSUE THREAD

**Issue by ecureuill** [dequelabs/axe-core#2507]
<pre><code>
axe-core version: 4.5.3
Chrome version:  84.0.4147.89 
Debian

</code></pre>

I'm facing a issue when running tests in a specific page.

For instance, when running the keyboard test, it stops as there is no more element in tab order whenever it tries to tab in the component bellow (code sample). 
Then, while I'm selecting the skipped elements as soon I click on any button of this component, the page freezes. Everything else works properly and no error prompts.

```html
<div class="content-item">
	<div class="draggable-item">
		<div class="content-box">
			<div class="activity-content-item">
				<div>
					<div class="activity-item-container">
						<div class="activity-container">
							<div class="MuiPaper-root MuiExpansionPanel-root activity-parent-panel Mui-expanded MuiExpansionPanel-rounded MuiPaper-elevation1 MuiPaper-rounded">
								<div class="MuiButtonBase-root MuiExpansionPanelSummary-root item-quiz-expansion-summary Mui-expanded" tabindex="0" role="button" aria-disabled="false" aria-expanded="true" aria-controls="panel1c-content" id="0.9625698419954549">
									<div class="MuiExpansionPanelSummary-content Mui-expanded">
										<div class="item-quiz-expansion-summary-text-container">
											<h2 class="activity-panel-title  MuiTypography-root activity-panel-title MuiTypography-body1">Activity</h2>
											<h3 class="quiz-panel-subtitle MuiTypography-root quiz-panel-subtitle MuiTypography-body1"><button class="MuiButtonBase-root MuiButton-root MuiButton-text quiz-panel-subtitle  MuiButton-textSizeLarge MuiButton-sizeLarge" tabindex="0" type="button" aria-expanded="true" aria-controls="sect1" id="acc1id"><span class="MuiButton-label">Storyboard</span><span class="MuiTouchRipple-root"></span></button></h3>
										</div>
									</div>
									<div class="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon Mui-expanded MuiIconButton-edgeEnd" aria-disabled="false" aria-hidden="true"><span class="MuiIconButton-label"><svg class="MuiSvgIcon-root" focusable="false" viewBox="0 0 24 24" aria-hidden="true">
												<path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"></path>
											</svg></span><span class="MuiTouchRipple-root"></span></div>
								</div>
								<div class="MuiCollapse-container MuiCollapse-entered" style="min-height: 0px;">
									<div class="MuiCollapse-wrapper">
										<div class="MuiCollapse-wrapperInner">
											<div aria-labelledby="0.9625698419954549" id="panel1c-content" role="region">
												<div class="MuiExpansionPanelDetails-root item-quiz-detail">
													<div class="item-quiz-detail-container">
														<p class="activity-instruction-title">Instructions:</p>
														<div class="activity-item-container-instruction">
															<p>To complete this activity, select a story.</p><br>
															<p>conte uma historia</p>
														</div>
														<div>
															<p class="activity-instruction-title">Story:</p>
															<div class="activity-item-container-instruction"><a target="_blank" href="/story#4WhcWYwPwya2HbzBZ" style="text-decoration: none;">test story</a></div>
														</div>
													</div>
												</div>
												<hr class="MuiDivider-root">
												<div class="MuiExpansionPanelActions-root quiz-item-actions MuiExpansionPanelActions-spacing">
													<div><button class="MuiButtonBase-root MuiButton-root MuiButton-text" tabindex="0" type="button"><span class="MuiButton-label">Set reminder</span><span class="MuiTouchRipple-root"></span></button><button class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary" tabindex="0" type="button"><span class="MuiButton-label">Do activity</span><span class="MuiTouchRipple-root"></span></button></div>
													<div class="align-items-center"><button class="MuiButtonBase-root MuiButton-root MuiButton-text" tabindex="0" type="button"><span class="MuiButton-label">Activity done</span><span class="MuiTouchRipple-root"></span></button><svg class="MuiSvgIcon-root done-icon" focusable="false" viewBox="0 0 24 24" aria-hidden="true">
															<path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"></path>
														</svg></div>
												</div>
											</div>
										</div>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>
```

**Comment by WilcoFiers** [comment]
Thank you for reporting. We'll take a look!

**Comment by erik-deque** [comment]
This was fixed in 4.7.1
